TLM/TMPE.CitiesGameBridge/Service/NetService.cs

```diff
@@ -121,6 +121,12 @@ public class NetService : INetService {
                     }
                 } else {
                     ushort segmentId = node.GetSegment(0);
+                    for (int i = 0; i < 8; ++i) {
+                        segmentId = node.GetSegment(i);
+                        if (segmentId != 0) {
+                            break;
+                        }
+                    }
```

Comment:
Remove the initial variable initialization. Just `ushort segmentId`.
Also handle the situation when all `segmentId`s in the node are 0 (in that case `segmentId` will also be 0). Could be a highly improbable case but what do i know about how the game works?